staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2277,7 +2277,7 @@ type ExecAction struct {
 type Probe struct {
 	// The action taken to determine the health of a container
 	ProbeHandler `json:",inline" protobuf:"bytes,1,opt,name=handler"`
-	// Number of seconds after the container has started before liveness probes are initiated.
+	// Number of seconds after the container has started before startup, liveness or readiness probes are initiated.
```

Comment:
I would actually recommend making this reference the parent field name less

```suggestion
	// Number of seconds after the container has started before the probe is initiated.
```